src/test/java/dk/kb/labsapi/ImageExportTest.java

```diff
@@ -44,6 +55,81 @@ static void setupConfig() throws IOException {
         ServiceConfig.initialize("conf/labsapi*.yaml");
     }
 
+    // Illustrative unit test for mocking static methods
+    @Test
+    void testStaticMocking() throws MalformedURLException {
+        // All mocking takes place inside the try-block
+        try (MockedStatic<ImageExport> mockedExport = Mockito.mockStatic(ImageExport.class)) {
+            // Setup static method mock
+            mockedExport.when(() -> ImageExport.setUsableEndYear(2040)).thenReturn(2023);
+
+            // Perform unit tests
+            assertEquals(2023, ImageExport.setUsableEndYear(2040),
+                    "Calling a mocked static with the correct input should return the expected output");
+
+            assertEquals(0, ImageExport.setUsableEndYear(2050),
+                    "Calling a mocked static with unsupported input should return null (= 0 as int is an Atomic)");
+        }
+    }
+
+    @Test
+    void testInstanceMocking() throws MalformedURLException {
+        // All mocking takes place inside the try-block
+        // We use Mockedconstruction as we want the standard constructor for ImageExport to be evaluated
+        try (MockedConstruction mocked = mockConstruction(ImageExport.class)) {
+            ImageExport mockedExport = ImageExport.getInstance();
+            when(mockedExport.convertPageUUID("foo")).thenReturn("bar", "bar2");
+            when(mockedExport.convertPageUUID("hello")).thenReturn("world");
+
+            // Perform unit tests using the mocked instance
+            assertEquals("bar", mockedExport.convertPageUUID("foo"),
+                    "Calling a mocked method with the first correct input should return the first expected output");
+            assertEquals("bar2", mockedExport.convertPageUUID("foo"),
+                    "Calling a mocked method a second time with the first correct input should return the second expected output");
+            assertEquals("bar2", mockedExport.convertPageUUID("foo"),
+                    "Calling a mocked method a third time with the first correct input should return the last output in the response chain");
+
+            assertEquals("world", mockedExport.convertPageUUID("hello"),
+                    "Calling a mocked method with the other correct input should return the expected output");
+
+            assertNull(mockedExport.convertPageUUID("zoo"),
+                    "Calling a mocked method with unexpected input should return null");
+        }
+    }
+
+
+    @Test
+    void testCsvConstructionWithMockedDownload() throws IOException {
+        StreamingOutput output = testOutput();
+        assertNotNull(output);
+
```

Comment:
Instead of only checking for `null`, it should be possible to check the content with something like this:
```
        ByteArrayOutputStream raw = new ByteArrayOutputStream();
        output.write(raw);
        String content = raw.toString(StandardCharsets.UTF_8);
        assertEquals("Image1", content);
```
The problem is that `content` seems to be the empty string!?